tests/test_project.py

```diff
@@ -107,12 +107,13 @@ def is_present(cls):
         return True
 
 
-class BaseProjectTest(unittest.TestCase):
+class TestBaseProject():
```

Comment:
This class name is misleading.

- [ ] Please rename `TestBaseProject` to `TestProjectBase`.